backend/coins/eth/account.go

```diff
@@ -147,6 +163,38 @@ func (account *Account) poll() {
 	}
 }
 
+// pendingOutgoingTransactions gets all locally stored pending outgoing transactions. It filters out
+// already confirmed ones.
+func (account *Account) pendingOutgoingTransactions(confirmedTxs []coin.Transaction) (
+	[]coin.Transaction, error) {
+	dbTx, err := account.db.Begin()
+	if err != nil {
+		return nil, err
+	}
+	defer dbTx.Rollback()
+	pendingOutgoingTransactions, err := dbTx.PendingOutgoingTransactions()
+	if err != nil {
+		return nil, err
+	}
+
+	confirmedTxHashes := map[string]struct{}{}
```

Comment:
Is `struct{}` preferable to `boolean`?

Reply:
yes. boolean contains useless information (true or false doesn't mean anything here).